products/drupal.md

```diff
@@ -25,22 +25,22 @@ releases:
 -   releaseCycle: "10.0"
     support: 2023-06-21
     eol: 2023-12-13
-    latest: "10.0.5"
-    latestReleaseDate: 2023-03-15
+    latest: "10.0.7"
```

Comment:
10.0.6 was released on the same day : https://www.drupal.org/project/drupal/releases/10.0.6. Same for 9.5.